react-ui-org/react-ui
# Unify API of form components (#28, #113)

- unify API (closes #28)
  - change `error` to `validationText`
  - change `helperText` and `description` to `helpText`
  - make sure both of the new props are supported in all supported components
- `.eslintrc` clean up and related changes (closes #113)
